teal-language-server-0.1.2-1.rockspec

```diff
@@ -21,15 +21,16 @@ build_dependencies = {
 
 dependencies = {
    "luafilesystem",
-   "tl == 0.24.4",
+   "tl == 0.24.5",
```

Comment:
Teal released `0.24.6` this week:

https://github.com/teal-language/tl/releases/tag/v0.24.6